server/activate.go

```diff
@@ -24,89 +23,105 @@ func (p *Plugin) OnActivate() error {
 		p.serverVersion = serverVersion
 	}
 
-	if err := p.ensureBotExists(); err != nil {
+	if botUserId, err := p.ensureBotExists(); err != nil {
```

Comment:
nit: idiomatic coding style would assign these outside the if statement so that lines 28-29 don't need to be inside one.